migrations/20161129114256-create-admin.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Admins', {
```

Comment:
You shouldnt have a Admin model because it duplicates data with User. Instead you should have a isAdmin column in your User table